### PR TITLE
A few edits

### PR DESCRIPTION
**Pull request by CBonnell**

### CODE REVIEW

File: docs/BR.md [hunk 1]
```diff
@@ -1099,9 +1101,20 @@ No stipulation.
...
+#### 4.3.1.2 Linting of to-be-signed Certificate content
+
+Due to the complexity in implementing the certificate-formatting standards of these Requirements when issuing publicly-trusted Certificates, before a Certificate or Pre-certificate is issued, the CA SHOULD implement a Linting process to test the technical conformity of the Certificate to be issued with these Requirements.
+
+Methods used to produce a certificate containing to-be-signed Certificate content include, but are not limited to:
+
+1. Sign the `tbsCertificate` with a "dummy" Private Key whose Public Key component is not certified by a Certificate that does not chain to a publicly-trusted CA Certificate; or
```
Comment (slghtr-says) [inline docs/BR.md:1114]:
```suggestion
1. Sign the `tbsCertificate` with a "dummy" Private Key whose Public Key component is certified by a Certificate that does not chain to a publicly-trusted CA Certificate; or
```
Removing 'not' to remove the double-negative

File: docs/BR.md [hunk 2]
```diff
@@ -1099,9 +1101,20 @@ No stipulation.
...
+#### 4.3.1.2 Linting of to-be-signed Certificate content
+
+Due to the complexity in implementing the certificate-formatting standards of these Requirements when issuing publicly-trusted Certificates, before a Certificate or Pre-certificate is issued, the CA SHOULD implement a Linting process to test the technical conformity of the Certificate to be issued with these Requirements.
+
+Methods used to produce a certificate containing to-be-signed Certificate content include, but are not limited to:
+
+1. Sign the `tbsCertificate` with a "dummy" Private Key whose Public Key component is certified by a Certificate that does not chain to a publicly-trusted CA Certificate; or
```
Comment (CBonnell) [inline docs/BR.md:1114]:
```suggestion
1. Sign the `tbsCertificate` with a "dummy" Private Key whose Public Key component is not certified by a Certificate that chains to a publicly-trusted CA Certificate; or
```

File: docs/BR.md [hunk 3]
```diff
@@ -382,6 +382,8 @@ The Definitions found in the CA/Browser Forum's Network and Certificate System S
 
 **Legal Entity**: An association, corporation, partnership, proprietorship, trust, government entity or other entity with legal standing in a country's legal system.
 
+**Linting**: A process where the content of a signed document such as a Certificate, Certificate Revocation List, or OCSP response is checked for conformance with the profile defined in these Requirements.
```
Comment (dzacharo) [inline docs/BR.md:385]:
```suggestion
**Linting**: A process where the content of digitally signed data such as a Certificate, Certificate Revocation List, or OCSP response, or data-to-be-signed like a `tbsCertificate` (as described in [section 4.1.1.1 of RFC 5280](https://datatracker.ietf.org/doc/html/rfc5280#section-4.1.1.1)) is checked for conformance with the profiles and requirements defined in these Requirements.
```

**Review by dzacharo — CHANGES_REQUESTED** [review]
Some improvements to the definition of "Linting". I like the new structure of 4.3.1.

**Pull request closed without merging**